src/client/js/otp/config.js

```diff
@@ -291,49 +291,25 @@ i18n.init(options, function(t) {
 
 });
 
+//TRANSLATORS: Travel by: mode of transport (Used in selection in Travel
+//Options widgets)
```

Comment:
Move this comment back to, and change the comment to:

`TRANSLATORS: Used in selection in Travel Options widgets`